src/test/java/net/fabricmc/test/V1ModJsonParsingTests.java

```diff
@@ -48,7 +48,7 @@ final class V1ModJsonParsingTests {
 	private static Path errorPath;
 
 	@BeforeAll
-	private static void setupPaths() {
+	public static void setupPaths() {
```

Comment:
Is this needed? I don't see it being used anywhere

Reply:
Updating JUnit caused this to start failing, it now requires test methods like this to be public.